src/librbd/cache/pwl/ReplicatedWriteLog.h

```diff
@@ -50,6 +50,43 @@ class ReplicatedWriteLog : public AbstractWriteLog<ImageCtxT> {
   using C_WriteSameRequestT = pwl::C_WriteSameRequest<This>;
   using C_CompAndWriteRequestT = pwl::C_CompAndWriteRequest<This>;
 
+  using AbstractWriteLog<ImageCtxT>::m_lock;
+  using AbstractWriteLog<ImageCtxT>::m_log_entries;
+  using AbstractWriteLog<ImageCtxT>::m_image_ctx;
+  using AbstractWriteLog<ImageCtxT>::m_perfcounter;
+  using AbstractWriteLog<ImageCtxT>::m_ops_to_flush;
+  using AbstractWriteLog<ImageCtxT>::m_cache_state;
+  using AbstractWriteLog<ImageCtxT>::m_first_free_entry;
+  using AbstractWriteLog<ImageCtxT>::m_first_valid_entry;
+
+  PMEMobjpool *m_log_pool = nullptr;
+
+  void process_work() override;
```

Comment:
Nit: these overrides are technically part of the protected section.

Reply:
Done